app/views/ferns/show.html.erb

```diff
@@ -8,41 +8,33 @@
 
   <div class='container'>
     <div class='row fern-buttons'>
-      <% if @fern.health <= 3 %>
-        <div class='col-sm-4'>
-          <%= button_to 'Water Fern', water_fern_path(@fern.id), method: :get, class: 'btn btn-lg btn-primary' %>
-        </div>
-        <div class='col-sm-4'>
-          <%= button_to 'Compost Fern', fern_path(@fern.id), method: :delete, class: 'btn btn-lg btn-primary', data: {confirm: "Are you sure you want to compost #{@fern.name}?"} %>
-        </div>
-        <div class='col-sm-4'>
-          <%= button_to 'Fertilize Fern', fertilize_fern_path(@fern.id), method: :get, class: 'btn btn-lg btn-primary' %>
-        </div>
-      <% else %>
-        <div class='col-sm-3'>
-        </div>
-        <div class='col-sm-3'>
-          <%= button_to 'Water Fern', water_fern_path(@fern.id), method: :get, class: 'btn btn-lg btn-primary' %>
-        </div>
-        <div class='col-sm-3'>
-          <%= button_to 'Compost Fern', fern_path(@fern.id), method: :delete, class: 'btn btn-lg btn-primary', data: {confirm: "Are you sure you want to compost #{@fern.name}?"} %>
-        </div>
-      <% end %>
+      <div class='col-sm-4'>
+        <%= button_to 'Water Fern', water_fern_path(@fern.id), method: :get, class: 'btn btn-lg btn-primary' %>
+      </div>
+      <div class='col-sm-4'>
+        <%= button_to 'Compost Fern', fern_path(@fern.id), method: :delete, class: 'btn btn-lg btn-primary', data: {confirm: "Are you sure you want to compost #{@fern.name}?"} %>
+      </div>
+      <div class='col-sm-4'>
+        <%= button_to 'Fertilize Fern', fertilize_fern_path(@fern.id), method: :get, class: 'btn btn-lg btn-primary' %>
+      </div>
     </div>
   </div>
   <div class="spacer-2"></div>
   <div class='container show-info'>
     <div class="row">
-      <div class="col-sm-6">
+      <div class="col-sm-5">
         <h4>Preferned Contact Method:</h4>
         <%= @fern.preferred_contact_method.capitalize %>
       </div>
-      <div class="col-sm-6">
+      <div class="col-sm-7 text-center">
         <div id="last_3_interactions">
           <h4>Last Three Interactions:</h4>
-          <% @fern.interactions.each do |interaction| %>
-            <%= interaction.display_message %><br>
-          <% end %>
+          <div style="display: inline-block; text-align: left;">
+            <% @fern.interactions.each do |interaction| %>
+              <%= image_tag(interaction.emoji, class: 'emoji') %>
```

Comment:
Hit an error here `the asset "emojis/.png" is not present in the asset pipeline`. I am running this locally, I might reset my db in case that is causing the issue.